src/volt/llvm/expression.d

```diff
@@ -1145,9 +1147,9 @@ void handleCall(State state, ir.Postfix postfix, Value result)
 		llvmArgs[0] = llvmExtraArg;
 	}
 
-	foreach (i, arg; postfix.arguments) {
+	for (size_t i = 0; i < postfix.arguments.length; ++i) {
```

Comment:
Any reason for this change? Seem like some debug code or early prototype change?